features/IFC001-official-schema-identifier.feature

```diff
@@ -0,0 +1,10 @@
+@implementer-agreement
+@IFC
+Feature: CRI - Official Schema Identifier
+This rule verifies that the IFC model conforms to one of the official versions released by buildingSMART.
```

Comment:
I can't commit the suggestion directly on Github, probably because the `git diff`is outdated.
Modified it in https://github.com/buildingSMART/ifc-gherkin-rules/pull/114/commits/84d9d824a834a02a000bc4fa27d479570e353a16